bsp_diff/caas/device/intel/civ/host/vm-manager/0046-Multi-Android-VM-support.patch

```diff
@@ -0,0 +1,407 @@
++
++	mv $USER_HOME/.intel/.civ/civ-sriov.ini $USER_HOME/.intel/.civ/$VM_NAME.ini
+ }
+ 
+ function copy_civ_ini() {
+@@ -39,6 +48,30 @@ function copy_civ_ini() {
+         chmod 0666 $USER_HOME/.intel/.civ/civ-sriov.ini
+ }
+ 
++function read_args() {
++        while [[ $# -gt 0 ]]; do
++                case $1 in
++                        -v)
++                                VSOCK_ID=$2
++                                shift
++                                ;;
++
++                        -p)
++                                ADB_PORT=$2
++                                shift
++                                ;;
++
++                        -n)
++                                VM_NAME=$2
++                                shift
++                                ;;
++                esac
++                shift
++        done
++}
++
++read_args "$@"
++
+ copy_civ_ini
+ setup_civ_ini
+ 
+diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
+index 0c1e0a1..9f7fccb 100755
+--- a/scripts/setup_host.sh
++++ b/scripts/setup_host.sh
+@@ -19,6 +19,10 @@ CIV_WORK_DIR=$(pwd)
+ CIV_GOP_DIR=$CIV_WORK_DIR/GOP_PKG
+ CIV_VERTICAl_DIR=$CIV_WORK_DIR/vertical_patches/host
+ VM_MANAGER_VERSION=v1.2.3
++VSOCK_ID=3
++VM_NAME=Android-CIV
++ADB_PORT=5555
++USER_HOME=$(getent passwd $SUDO_USER | cut -d: -f6)
+ 
+ #---------      Functions    -------------------
+ function error() {
+@@ -575,6 +579,21 @@ function parse_arg() {
+                 setup_remote_infer || return -1
+                 ;;
+ 
++	    -v)
++                VSOCK_ID=$2
++		shift
++		;;
++
++	    -b)
++                ADB_PORT=$2
++		shift
++                ;;
++
++            -n)
++                VM_NAME=$2
++		shift
++                ;;
++
+             --auto-start)
+                 install_auto_start_service "$2" || return -1
+                 shift
+@@ -599,13 +618,37 @@ function parse_arg() {
+ }
+ 
+ function setup_civ_ini() {
+-        $CIV_WORK_DIR/scripts/setup_civ_ini.sh
++        $CIV_WORK_DIR/scripts/setup_civ_ini.sh -v $1 -p $2 -n $3
+ }
+ 
+ function create_aaf_dir() {
+         mkdir -p $CIV_WORK_DIR/scripts/aaf
+ }
+ 
++function create_vm_dir() {
++        if [ -e $CIV_DIR/$VM_NAME ]
++        then
++                echo "Folder with name $VM_NAME already present. Delete and create new folder? Please enter yes/no"
++		read input
++		if [ $input = "yes" ]; then
++	                rm -rf $CIV_DIR/$VM_NAME
++	        else
++	                exit
++	        fi
++        fi
++        echo "Dir: $CIV_WORK_DIR/$VM_NAME"
++        mkdir $CIV_WORK_DIR/$VM_NAME
++}
++
++
++function copy_files_for_vm() {
++    echo "Copying file: $CIV_WORK_DIR/$VM_NAME"
++    mkdir $CIV_WORK_DIR/$VM_NAME/scripts
++    cp -r $CIV_WORK_DIR/scripts/aaf $CIV_WORK_DIR/$VM_NAME/scripts/
++    cp OVMF.fd $CIV_WORK_DIR/$VM_NAME/
++    ls -la $CIV_WORK_DIR/$VM_NAME/
++    cp $CIV_WORK_DIR/scripts/rpmb_dev $CIV_WORK_DIR/$VM_NAME/scripts/
```

Comment:
For the three copy operations here, check that the files exists... otherwise display error and exit